models/inception_v4/model.py

```diff
@@ -0,0 +1,385 @@
+""" Pytorch Inception-V4 implementation
+Sourced from https://github.com/Cadene/pretrained-models.pytorch (MIT License)
+"""
+
+from __future__ import print_function, division, absolute_import
```

Comment:
We are not giving py2.7 support in other files, so do you think we should give it here?

Reply:
No we shouldn't.

Reply:
No i dont think so. removed it